build.gradle

```diff
@@ -599,5 +616,7 @@ task jacocoRootReport(type: JacocoReport, group: 'Coverage reports') {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.0'
+    gradleVersion = '3.2.1'
+    // We want the sources with the wrapper.
+    distributionType = DistributionType.ALL
```

Comment:
This should finally fix intellij constantly running the wrapper task and changing the wrapper properties